utilities/run_unit_tests.sh

```diff
@@ -19,8 +19,6 @@ validate_platform () {
     fi
 }
 
-set -e
```

Comment:
instead of deleting this line, we can also use ` || true` to skip or ignore unwanted errors